src/controllers/dynakube/dtclient_reconciler.go

```diff
@@ -146,11 +146,17 @@ func (r *DynatraceClientReconciler) Reconcile(ctx context.Context, instance *dyn
 
 	if instance.KubernetesMonitoringMode() &&
 		instance.FeatureAutomaticKubernetesApiMonitoring() {
-		tokens[0].Scopes = append(tokens[0].Scopes, dtclient.TokenScopeEntitiesRead, dtclient.TokenScopeEntitiesWrite)
+		tokens[0].Scopes = append(tokens[0].Scopes, dtclient.TokenScopeEntitiesRead)
 	}
 
-	if instance.IsActiveGateMode(dynatracev1beta1.MetricsIngestCapability.DisplayName) {
-		tokens[0].Scopes = append(tokens[0].Scopes, dtclient.TokenScopeMetricsIngest)
+	if instance.IsActiveGateMode(dynatracev1beta1.MetricsIngestCapability.DisplayName) && r.DataIngestToken != "" {
```

Comment:
```suggestion
	if r.DataIngestToken != "" {
```
should give a warning that the token is missing scope, but not block creation of the ActiveGate!

Reply:
why would this block the creation of the activeGate ?

Reply:
it doesn't, I confused it with a different error :slightly_smiling_face: